www/templates/builder.html

```diff
@@ -31,7 +31,17 @@
     <div class="head-2">{{ description }}</div>
 
     {% endif %}
-     
+
+      <h2 class="head-2">
```

Comment:
One finding, what about a situation, when there won't tags? Should we still display this header?

Reply:
Probably no

Reply:
Done